.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10
+    environment:
+      CLUSTER_CTL_VERSION: 0.8.0
+    steps:
+      - checkout
+      - run:
+          name: Build the optimized site to ./public
+          command: |
+            make
+      - run:
+          name: Install ipfs-cluster-ctl binary
+          command: |
+            wget https://dist.ipfs.io/ipfs-cluster-ctl/v${CLUSTER_CTL_VERSION}/ipfs-cluster-ctl_v${CLUSTER_CTL_VERSION}_linux-amd64.tar.gz
+            tar xvfz ipfs-cluster-ctl_v${CLUSTER_CTL_VERSION}_linux-amd64.tar.gz
+            sudo mv ipfs-cluster-ctl/ipfs-cluster-ctl /usr/local/bin
+      - run:
+          name: Download dnslink script
+          command: |
+            wget https://raw.githubusercontent.com/protocol/infra-team/master/website_deploys/dnslink.sh
+            sudo chmod +x dnslink.sh
+      - run:
+          name: Pin website, post notification for PRs or update DNS on master
+          command: |
+            hash=$(ipfs-cluster-ctl --host /dnsaddr/cluster.ipfs.io \
+                                    --basic-auth $CLUSTER_USER:$CLUSTER_PASSWORD \
+                                    add --rmin 3 --rmax 3 --name "$DNSIMPLE_ZONE" \
+                                    --recursive public/ | tail -n1 | cut -d " " -f 2 )
```

Comment:
note to self, the directory to use as the production ready webroot should be set via a per project env var. This project uses `dist`, and `public` is already used for artefacts that will end up in dist. cc @scout